src/main.rs

```diff
@@ -238,9 +239,23 @@ fn compile_sass(filename: &str) {
         .expect(&format!("couldn't write css file: {}", &css_file));
 }
 
+fn concat_vendor_css(files: Vec<&str>) {
+    let mut concatted = String::new();
+    for filestem in files {
+        let vendor_path = format!("./static/styles/{}.css", filestem);
+        let mut file = File::open(vendor_path).expect("couldn't read vendor css");
```

Comment:
Certainly not blocking on this, just a nice trick: for this exact pattern you can use [`fs::read_to_string`](https://doc.rust-lang.org/1.30.1/std/fs/fn.read_to_string.html) instead:

```rust
let contents = fs::read_to_string(vendor_path).expect("couldn't read vendor css");
concatted.push_str(&contents);
```